Blog post about Digital Transformation (#117)
simplificator/simplificator-blog: content/posts/die-digitale-transformation-ist-eine-reise.md

language: de
author: Shouq Hussein
---

Die Welt verändert sich, und die Unternehmen müssen sich mit ihr verändern. Es reicht nicht mehr aus, dass sich die Unternehmen auf ihre traditionellen Geschäftsmodelle konzentrieren. Sie müssen sich auch der digitalen Transformation stellen, wenn sie in der neuen digitalen Wirtschaft überleben wollen. Aber was genau bedeutet "digitale Transformation" und wie können Unternehmen sie erreichen? Hier sind einige nützliche Tipps, wie Innovationen die digitale Transformation unterstützen können:

## Die digitale Transformation ist eine Herausforderung

Der Wandel betrifft Menschen, Kultur und Prozesse. Bei der digitalen Transformation geht es nicht nur um Technologie. Es geht darum, wie Sie Ihr Unternehmen so verändern, dass es in Zukunft kundenorientierter und agiler wird. Um dieses Ziel zu erreichen, müssen Sie verstehen, welchen Nutzen Ihr Unternehmen aus der digitalen Transformation zieht und wo es derzeit steht.
Dies kann durch die Betrachtung von vier Dimensionen erreicht werden: 
Prozesse (Modernisieren Sie Legacy-Prozesse, beschleunigen Sie effiziente Workflows, stärken Sie die Sicherheit und steigern Sie Ihre Produktivität)

> Menschen (Übergang von einzelnen Mitarbeitern zu vernetzten Teams)
> Kultur (Übergang von einer Befehls- und Kontrollkultur zu einer gemeinsamen Identität)
> Technologie (Übergang von monolithischen IT-Systemen zu Microservices)

Reviewer: ```suggestion

* Menschen (Übergang von einzelnen Mitarbeitern zu vernetzten Teams)
* Kultur (Übergang von einer Befehls- und Kontrollkultur zu einer gemeinsamen Identität)
* Technologie (Übergang von monolithischen IT-Systemen zu Microservices)

```

## Lösen Sie das richtige Problem

> Der erste Schritt bei jedem Projekt zur digitalen Transformation besteht darin, das Problem, das Sie lösen wollen, zu definieren und nicht einfach mit einer Idee zu beginnen. Deshalb ist es wichtig, dass Sie Ihr Team zusammenbringen und ihm Zeit geben, ein Brainstorming über die Art der Probleme zu machen, mit denen es konfrontiert ist. Danach gilt es herauszufinden, wie Technologie zur Lösung dieser Probleme beitragen könnte. Der Schlüssel dazu ist die Einsicht, dass es bei der digitalen Transformation nicht darum geht, Menschen zu ersetzen, sondern darum, menschliche Fähigkeiten zu entwickeln und zu verbessern. Besonders am Anfang lassen sich zum Teil mit einfachen Schritten grosse Vereinfachungen erreichen.

Reviewer: I think all other articles use informal tenses (Du). I don't particularly mind, but I think the web site and blog should have the same approach to talking with people. What do you think?

Author: we agreed to keep it in Sie format

Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung von KI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.

## Finden Sie die richtigen Technologie- und Lösungspartner

Sie werden Technologie- und Lösungspartner finden wollen, die Ihre Strategie der digitalen Transformation unterstützen können. Im Folgenden finden Sie einige Tipps, wie Sie dies tun können:

> Verstehen Sie die Vorteile der Zusammenarbeit mit einem Technologiepartner: Ein Partner kann Ihrem Unternehmen helfen, Zugang zu den neuesten Spitzentechnologien zu erhalten, aber er verfügt auch über umfassende Erfahrung mit der Implementierung neuer Lösungen, die sich in anderen Unternehmen bewährt haben. 

Reviewer: If you want to make paragraphs, you need an empty line inbetween. There's a preview option top right

Reviewer: Or even stars for a bullet list (as above)

## Testen und pilotieren Sie neue Technologien, aber nur dort, wo es sinnvoll ist

Die digitale Transformation ist kein Projekt, das für alle passt. Sie erfordert eine sorgfältige Berücksichtigung des Kontexts und der Kultur Ihres Unternehmens sowie der Bedürfnisse und Fähigkeiten Ihrer Mitarbeiter. Ihre Lösung kann nicht einfach lauten: "Wir setzen Technologie ein, um dieses Problem zu lösen." Sie müssen sich Gedanken darüber machen, welche Art von Technologie für Ihre spezielle Situation am besten geeignet ist, wer sie nutzen wird und wie diese Personen mit ihr interagieren werden.
Wir arbeiten mit Unternehmen zusammen, um wirkungsvolle Anwendungsfälle zu identifizieren, zu priorisieren und umzusetzen. Auf diese Weise können Kunden schnell Werte schaffen und gleichzeitig demonstrieren, was digitale Technologien und Arbeitsweisen für das Unternehmen leisten können. Der Prozess umfasst drei Phasen: 

> Innovation (Erstellung eines Machbarkeitsnachweises und Validierung des Geschäftsszenarios).

Author: *

> Inkubation (Einführung eines minimal realisierbaren Produkts und Testen und Lernen mit agilen Sprints).

Author: *

> Industrialisierung (Ausführen der Technologie und des Geschäftsprozesses in grossem Massstab). 

Author: *

Nach einer ersten Welle von Anwendungsfällen wiederholt sich der Prozess mit der nächsten Welle – und so weiter. Was in den verschiedenen Phasen umgesetzt wird ist nicht nur eine Frage des gesunden Menschenverstandes, sondern auch entscheidend, um die Zustimmung Ihres Teams zu erhalten und die Kosten überschaubar zu halten. Wenn Sie neue Technologien ausprobieren ohne zunächst zu verstehen, warum sie für Ihr spezielles Geschäftsproblem oder Ihre Geschäftseinheit geeignet sind, könnten Sie viel mehr als nötig in Testversionen investieren, die nie von jemandem im Team verwendet werden (und daher nutzlos sind). Andererseits sollten Sie, wenn Sie bahnbrechende Innovationen wie Chatbots oder künstliche Intelligenz (KI) testen, nicht einfach das für bare Münze nehmen, was deren Anbieter Ihnen sagen. 

> ## Arbeiten Sie Agile

Reviewer: ```suggestion
## Arbeiten Sie Agil
```

## Vergessen Sie nicht die Menschen und die Kultur

Dies ist der wichtigste Aspekt der Innovation, insbesondere wenn es um die digitale Transformation geht. Menschen sind nicht nur eine Ressource, sondern auch ein Vermögenswert. Der Erfolg eines Unternehmens hängt von seinen Mitarbeitern ab: Sie sind die Hauptakteure bei der Gestaltung der Unternehmenskultur und tragen zum Erfolg oder Misserfolg des Unternehmens bei. Sie können noch so tolle Ideen haben, wenn Sie kein Team haben, mit dem sie umgesetzt werden können, werden sie nicht zum Tragen kommen.

> Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation ist eine Reise, kein Ziel.

Reviewer: ```suggestion
Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation selbst ist eine Reise, kein Ziel an sich.
```
I find it confusing to first say you can reach your goals, and then that transformation is not a goal but a path. I think my suggestion makes that clearer.

Prozesse (Modernisieren Sie Legacy-Prozesse, beschleunigen Sie effiziente Workflows, stärken Sie die Sicherheit und steigern Sie Ihre Produktivität)

* Menschen (Übergang von einzelnen Mitarbeitern zu vernetzten Teams)
* Kultur (Übergang von einer Befehls- und Kontrollkultur zu einer gemeinsamen Identität)
* Technologie (Übergang von monolithischen IT-Systemen zu Microservices)

## Lösen Sie das richtige Problem

Der erste Schritt bei jedem Projekt zur digitalen Transformation besteht darin, das Problem, das Sie lösen wollen, zu definieren und nicht einfach mit einer Idee zu beginnen. Deshalb ist es wichtig, dass Sie Ihr Team zusammenbringen und ihm Zeit geben, ein Brainstorming über die Art der Probleme zu machen, mit denen es konfrontiert ist. Danach gilt es herauszufinden, wie Technologie zur Lösung dieser Probleme beitragen könnte. Der Schlüssel dazu ist die Einsicht, dass es bei der digitalen Transformation nicht darum geht, Menschen zu ersetzen, sondern darum, menschliche Fähigkeiten zu entwickeln und zu verbessern. Besonders am Anfang lassen sich zum Teil mit einfachen Schritten grosse Vereinfachungen erreichen.

> Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung von KI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.

Reviewer 2: ```suggestion
Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung einer komplexen Technologie wie AI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.
```

## Finden Sie die richtigen Technologie- und Lösungspartner

Sie werden Technologie- und Lösungspartner finden wollen, die Ihre Strategie der digitalen Transformation unterstützen können. Im Folgenden finden Sie einige Tipps, wie Sie dies tun können:

> Verstehen Sie die Vorteile der Zusammenarbeit mit einem Technologiepartner: Ein Partner kann Ihrem Unternehmen helfen, Zugang zu den neuesten Spitzentechnologien zu erhalten, aber er verfügt auch über umfassende Erfahrung mit der Implementierung neuer Lösungen, die sich in anderen Unternehmen bewährt haben. 
> Finden Sie die wichtigsten Kriterien für die Auswahl eines Partners: Stellen Sie sicher, dass Sie klare Anforderungen an die einzelnen Anbieter haben, bevor Sie sich für einen Anbieter entscheiden. So werden Sie später keine Überraschungen erleben, wenn Sie Zeit damit verbringen, die Produkte oder Dienstleistungen der anderen Anbieter zu bewerten.
> Erstellen Sie eine umfassende Liste potenzieller Anbieter auf der Grundlage dieser Kriterien
> Anbieter bewerten: Nehmen Sie Kontakt zu den Anbietern auf und bitten Sie sie um detaillierte Informationen über ihre Produkte oder Dienstleistungen sowie um Fallstudien von anderen Kunden, die sie erfolgreich eingesetzt haben. * Arbeiten Sie mit dem besten Anbieter für Ihre Bedürfnisse zusammen: Wenn Sie den richtigen Partner gefunden haben, ist es an der Zeit, mit ihm einen Plan für die Einführung der neuen Technologie zu erarbeiten.
> Lassen Sie sich nicht zu viel verkaufen. Lösungspartner leben davon, die Probleme als grösser darzustellen als sie sind, und vertrauen darauf dass ihre Kunden nicht genug technisches Selbstbewusstsein haben.

Reviewer 2: would good to add a list here:

```suggestion
* Verstehen Sie die Vorteile der Zusammenarbeit mit einem Technologiepartner: Ein Partner kann Ihrem Unternehmen helfen, Zugang zu den neuesten Spitzentechnologien zu erhalten, aber er verfügt auch über umfassende Erfahrung mit der Implementierung neuer Lösungen, die sich in anderen Unternehmen bewährt haben. 
* Finden Sie die wichtigsten Kriterien für die Auswahl eines Partners: Stellen Sie sicher, dass Sie klare Anforderungen an die einzelnen Anbieter haben, bevor Sie sich für einen Anbieter entscheiden. So werden Sie später keine Überraschungen erleben, wenn Sie Zeit damit verbringen, die Produkte oder Dienstleistungen der anderen Anbieter zu bewerten.
* Erstellen Sie eine umfassende Liste potenzieller Anbieter auf der Grundlage dieser Kriterien
* Anbieter bewerten: Nehmen Sie Kontakt zu den Anbietern auf und bitten Sie sie um detaillierte Informationen über ihre Produkte oder Dienstleistungen sowie um Fallstudien von anderen Kunden, die sie erfolgreich eingesetzt haben.
* Arbeiten Sie mit dem besten Anbieter für Ihre Bedürfnisse zusammen: Wenn Sie den richtigen Partner gefunden haben, ist es an der Zeit, mit ihm einen Plan für die Einführung der neuen Technologie zu erarbeiten.
* Lassen Sie sich nicht zu viel verkaufen. Lösungspartner leben davon, die Probleme als grösser darzustellen als sie sind, und vertrauen darauf dass ihre Kunden nicht genug technisches Selbstbewusstsein haben.
```

## Trends und Innovationen pragmatisch bewerten

> Um Ihr Unternehmen auf dem Weg der digitalen Transformation zu begleiten ist es wichtig zu verstehen, wie Trends und Innovationen helfen können. Der erste Schritt besteht darin, diese Entwicklungen aus einem pragmatischen Blickwinkel heraus zu bewerten. Sie müssen nicht nur die Technologie, sondern auch den Business Case, die Personalprozesse und die Auswirkungen auf die Kultur betrachten. Ihr Ziel sollte es sein, eine Lösung zu finden, die mit Ihrer Vision für die Zukunft übereinstimmt und gleichzeitig die aktuellen Anforderungen innerhalb eines angemessenen Zeitrahmens erfüllt.

Reviewer 2: ```suggestion
Um Ihr Unternehmen auf dem Weg der digitalen Transformation zu begleiten, ist es wichtig zu verstehen, wie Trends und Innovationen helfen können. Der erste Schritt besteht darin, diese Entwicklungen aus einem pragmatischen Blickwinkel heraus zu bewerten. Sie müssen nicht nur die Technologie, sondern auch den Business Case, die Personalprozesse und die Auswirkungen auf die Kultur betrachten. Ihr Ziel sollte es sein, eine Lösung zu finden, die mit Ihrer Vision für die Zukunft übereinstimmt und gleichzeitig die aktuellen Anforderungen innerhalb eines angemessenen Zeitrahmens erfüllt.
```
